src/psyclone/tests/psyir/transformations/extract_test.py

```diff
@@ -35,25 +35,33 @@
 # Modified by A. R. Porter, STFC Daresbury Lab
 # -----------------------------------------------------------------------------
 
-''' Module containing tests for PSyclone LFRicExtractTrans and
-GOceanExtractTrans transformations and ExtractNode.
+''' Module containing tests for PSyclone LFRicExtractTrans
```

Comment:
If this is now all LFRic specific then I think we need to move the file, e.g. to `tests/domain/lfric`

Reply:
It wasn't, but I've split this file into two, and moved the lfric-specific ones to tests/domain/lfric/transformations/lfric_extract_test.py